<?xml version="1.0" ?>
<launch>
    <param name="baseline" type="double" value="3.1"/>
    <param name="diameter" type="double" value="0.2"/>
    <param name="delta_x" type="double" value="0.7"/>
    <param name="delta_y" type="double" value="1.5"/>
    <param name="delta_theta" type="double" value="0"/>
    
    <node name="publisher" pkg="hausuebung1_semjon_eschweiler" output="screen" type="publisher">
        <remap from="/publisher" to="/robot"/>
    </node>
</launch>